imgui-glium-renderer/src/lib.rs

```diff
@@ -128,12 +132,17 @@ impl Renderer {
         self.device_objects
             .upload_index_buffer(&self.ctx, draw_list.idx_buffer)?;
 
-        let font_texture_id = self.device_objects.texture.get_id() as usize;
-
         let mut idx_start = 0;
         for cmd in draw_list.cmd_buffer {
-            // We don't support custom textures...yet!
-            assert!(cmd.texture_id as usize == font_texture_id);
+            let texture = match self.device_objects.textures.get(cmd.texture_id.into()) {
+                Some(tex) => tex,
+                // if an invalid ID is supplied, fall back to the font
+                None => match self.device_objects.textures.get(0.into()) {
```

Comment:
I think it would be nicer to return an error if an invalid ID is seen, because it's a clear programming error that should be fixed. We already have `RendererError` so it should be easy to just add another variant there